code/lib/manager-api/src/modules/shortcuts.ts

```diff
@@ -320,6 +322,10 @@ export const init: ModuleFn = ({ store, fullAPI }) => {
           fullAPI.expandAll();
           break;
         }
+        case 'remount': {
+          fullAPI.remount();
```

Comment:
I think there's no need for this to be a module in `api`. 

I suggest this just emits, the event directly:
```ts
fullAPI.emit(FORCE_REMOUNT, { storyId });
```

The state of "is the icon spinning" can be kept local, considering it's only used in 1 place, right?

The shortcut can simply emit the same event too, no need to have a public facing API in the `manager-api` package.

Reply:
I went with adding it as a module for two reasons:
- If I use `fullAPI.emit(FORCE_REMOUNT, { storyId });` in the shortcuts, I couldn’t figure out a way to still activate the animation if the button is the one keeping the animation state, unless that is fine? Also yes the animation state is only kept in one place 
-  This way it would allow for future shortcuts to be added easily to the other toolbar tools if needed and keep the structure similar to how the other shortcuts were setup

Reply:
> If I use fullAPI.emit(FORCE_REMOUNT, { storyId }); in the shortcuts, I couldn’t figure out a way to still activate the animation

The button/(code around it) can listen on the channel, and run the animation when the event is triggered?

I also don't really see the point in the updating the state after the animation is done playing.
I see that right now we remove the state to actually end the animation.

I think we could use a useEffect instead and a ref, then use code like this:
```
document.getElementById('logo').animate(
    [
        {},
        {
            filter: 'blur(10px)',
            transform: 'rotate(-15deg)',
            box-shadow: '0px 0px 10px 3px',
        },
        {
            height: '100px',
            width: '500px',
            left: '10px',
            top: '10px',
            box-shadow: '0px 0px 15px 5px rgba(0, 0, 0, 0.7)',
            background-color: 'rgba(0, 0, 1, 0.3)',
            opacity: '0.7',
        },
    ],
    {
        duration: 2000,
        delay: 2000,
        fill: 'forwards',
    },
)
```
see: https://stackoverflow.com/questions/44846614/trigger-css-animations-in-javascript

Reply:
> This way it would allow for future shortcuts to be added easily to the other toolbar tools if needed and keep the structure similar to how the other shortcuts were setup

Sure, if that's anticipated, but I'm not sure it is, because as I see it, there's no reason to even have this state at all.
We don't want to know if the animation is playing.. we don't use this state for anything. The state is to remove the animation after it's done playing... which is absolutely an internal state thing, not something that we'd want to manage in an application global place.

Reply:
Okay I will work on updating the flow👍

Reply:
@ndelangen Can you take a look at the PR again? I removed all the extra module created and just added a listener to the channel. Thank you for that tip! I wasn't aware of the listeners.